var/spack/repos/builtin/packages/conduit/package.py

```diff
@@ -209,9 +213,9 @@ def create_host_config(self, spec, prefix):
                 raise RuntimeError(msg)
             cmake_exe = cmake_exe.path
 
-        host_cfg_fname = "%s-%s-%s.cmake" % (socket.gethostname(),
-                                             sys_type,
-                                             spec.compiler)
+        host_cfg_fname = "%s-%s-%s-conduit.cmake" % (socket.gethostname(),
```

Comment:
Was this rename required? Or was it just cosmetic?

Reply:
w/o the rename, I can't spack activate those packages at the same time (files with the same name conflict)